noir/noir-repo/docs/docs/how_to/how-to-oracles.md

```diff
@@ -141,10 +141,10 @@ server.addMethod("resolve_function_call", async (params) => {
   if params.function !== "getSqrt" {
     throw Error("Unexpected foreign call")
   };
-  const values = params.inputs[0].Array.map((field) => {
```

Comment:
Went a little bit crazy debugging TXE after changing to session_ids, turns out this: https://github.com/noir-lang/noir/pull/5205/files#diff-171371e0d0df141bfd50a4404caf6cadd397fc78d575569d489936c67bb1a613R6 was breaking our typescript (de)serialization. All is good now, but docs were not updated correctly...I think they're good now, but a little bit unsure about the noir.js part CC/ @TomAFrench